src/Symfony/Component/Validator/Tests/Constraints/FileTest.php

```diff
@@ -26,7 +26,7 @@ public function testMaxSize($maxSize, $bytes, $binaryFormat)
     {
         $file = new File(['maxSize' => $maxSize]);
 
-        $this->assertSame($bytes, $file->maxSize);
+        $this->assertEquals($bytes, $file->maxSize);
```

Comment:
This test fails on Appveyor when dealing with the "4G" and "4g" datasets. I suspect PHP to cast 4 billion to a string when running on a 32bits architecture

Reply:
Can we make it 3G instead, to fit on 32bits ?

Reply:
This wouldn't fit. But I set it to 2G to fit signed integers.

Another solution would be to use `assertEquals` and `assertSame` depending on the architecture but I don't really like it